### ISSUE TITLE
[2.1.1] Service Discovery logs not being output

### ISSUE THREAD

**Issue by tkam8** [F5Networks/k8s-bigip-ctlr#1527]
#### Setup Details
CIS Version : 2.1.1
Build: f5networks/k8s-bigip-ctlr:latest
BIGIP Version: Big IP 14.1.0.1.0.0.7
AS3 Version: 3.22.1
Agent Mode: AS3
Orchestration: K8S
Orchestration Version: v1.18.6
Pool Mode: Nodeport
#### Description  

In older versions of CIS, "Discovered members" logs appeared in the bigip controller logs. 
v2.1.1 doesn't have these logs and it's used this for confirmation of service discovery.
Possible to enable this log to be output ? 

#### Steps To Reproduce
1) Deploy v2.1.1 with log level DEBUG
2) Deploy service and configmap
3) Watch logs (kubectl logs <bigip-ctlr> -n kube-system) and note no logs for service discovery

#### Expected Result 
```
2020/10/05 00:22:50 [DEBUG] [AS3] Discovered members for service nginx-app is [{240b:c1:a:b:c:1:1:1 30200 user-enabled} {240b:c1:a:b:c:1:1:2 30200 user-enabled} {240b:c1:a:b:c:1:1:3 30200 user-enabled} {240b:c1:a:b:c:1:1:4 30200 user-enabled} {240b:c1:a:b:c:1:1:5 30200 user-enabled} ]
```
#### Actual Result  

logs do not appear

**Comment by mdditt2000** [comment]
Checking with product development of this log

**Comment by tkam8** [comment]
Hi there, any initial response on these logs? Thanks

**Comment by mdditt2000** [comment]
This log was there up to 1.14 and got removed in 2.0. On my list of question for today's meeting as why it was removed. Maybe performance

**Comment by mdditt2000** [comment]
Filed PM Jira for internal tracking. Requesting for CIS 2.1 but cannot confirm until PD sprint planning

**Comment by mdditt2000** [comment]
Resolved in CIS 2.2.1. Closing.
